react-function-components/src/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
+
+const root = ReactDOM.createRoot(document.querySelector('#root'));
+const element = < CustomButton />;
+root.render(element);
```

Comment:
This is okay, but you can skip creating the `element` `const` and directly pass the component to `render`. For example:
```JSX
root.render(<CustomButton />);
```

Reply:
@scbowler Got it! That makes more sense.